sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -95,7 +94,12 @@
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-servicebus-jms-spring-boot-starter</artifactId>
-        <version>${azure.spring.boot.jms.integration.version}</version>
+        <version>${azure.spring.boot.integration.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>spring-starter-azure-storage</artifactId>
+        <version>${azure.spring.boot.integration.version}</version>
```

Comment:
This version should be different from the one of starters?

Reply:
done. updated